CampaignConfig/mdc2020_corsika.cfg

```diff
@@ -1,18 +1,19 @@
 [global]
-includes = /exp/mu2e/app/users/oksuzian/muse_101323/Production/CampaignConfig/mdc2020_prolog.cfg
-#includes = /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020ae/Production/CampaignConfig/mdc2020_prolog.cfg
+includes = /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020ae/Production/CampaignConfig/mdc2020_prolog.cfg
 
 db_folder = mdc2020ab
 run_number = 1200
 release = MDC2020
 release_v_i = ab
-release_v_o = ad
+release_v_o = ae
 njobs_s1 = 17500
-njobs = 10
-events_per_job = 5000
+njobs = 82330
+events_per_job = 500000
 s1_start_job=0
-s1type = Low # Only accepts 'Low' or 'All'
+#s1type only accepts 'Low' or 'All'
+s1type = Low 
```

Comment:
Is s1type still used?  It looks instead as if both Low and All output streams are always defined.